From 70bfd05fa89a9910783f57b5ab2f1230585c4c29 Mon Sep 17 00:00:00 2001
From: syphyr <syphyr@gmail.com>
Date: Thu, 10 Jan 2019 01:09:14 +0100
Subject: [PATCH] Bump Security Patch Level to 2019-01-05

Implemented:
============

CVE-2018-9583	A-112860487	RCE	Critical	7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2018-9584	A-114047681	EoP	High		7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2018-9585	A-117554809	EoP	High		7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2018-9586   A-116754444     EoP     High            7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2018-9587	A-113597344	EoP	High		7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2018-9588	A-111450156	ID	High		7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2018-9589	A-111893132	ID	High		7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2018-9590	A-115900043	ID	High		7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2018-9591	A-116108738	ID	High		7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2018-9592	A-116319076	ID	High		7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2018-9593	A-116722267	ID	High		7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2018-9594	A-116791157	ID	High		7.0, 7.1.1, 7.1.2, 8.0, 8.1, 9

No impact:
==========

CVE-2018-9582	A-112031362	EoP	High		8.0, 8.1, 9

Change-Id: Ie423b66eb0059520f414ab2ca77e1fadcd9314c3
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index a908f7d..546e6fb 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -131,7 +131,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2018-12-05
+    PLATFORM_SECURITY_PATCH := 2019-01-05
 endif
 
 ifeq "" "$(PLATFORM_BASE_OS)"
-- 
2.7.4

